Jenkinsfile.integration

```diff
@@ -32,6 +32,7 @@ pipeline {
            as the names of the heat stacks will be derived from this. Also
            the CaaSP Velum automation has issues with mixed case hostnames. */
         SOCOK8S_ENVNAME = "cloud-socok8s-${env.BRANCH_NAME.replaceAll("[^a-zA-Z0-9-]+", "-").toLowerCase()}-${env.BUILD_NUMBER}"
+        SOCOK8S_USE_PRV_MIRROR="true"
```

Comment:
Could we allow a custom mirror url instead of this bool var?

Reply:
Yeah, let's try that ...